### PR TITLE
[v1 API Docs] Add css for bulk tags, plus some quick fixes

### PR DESCRIPTION
**Pull request by juliawu**
Convert [bulk] on api/rest/v1/index.md to bulk tags that render with CSS.

Also corrects some minor formatting errors in the getting started guide.

Screen Shot 2022-09-09 at 12 51 03 PM

### CODE REVIEW

File: api/rest/v1/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ https://api.datacommons.org
 ## Simple vs Bulk Query
```
Comment (shifucun) [inline api/rest/v1/index.md:33]: Should all these titles use "###" instead of "##"?

**Review by shifucun — APPROVED** [review]
looks very nice! Now I know how to add a css tag :)

**Comment by shifucun** [comment]
just wonder if the v0 folder change affects current deployment? anywhere else need to be updated to refer to the files?

**Comment by juliawu** [comment]
It doesn't change the navigation behavior, but does change the link. Reverted the v0 folder change for now.

**Review by shifucun — APPROVED** [review]

**Pull request merged**